layouts/partials/footer.html

```diff
@@ -38,7 +38,11 @@
         <div class="footer-row text-center">
             <a class="footer-item" href="/cudf-pandas"><i class="fa-light fa-gauge-circle-bolt"></i> cuDF pandas</a>
             <span class="footer-item"> | </span>
+            <a class="footer-item" href="/cuvs"><i class="fa-light fa-chart-scatter-3d"></i> cuVS</a>
+            <span class="footer-item"> | </span>
             <a class="footer-item" href="/polars-gpu-engine"><i class="fa-light fa-igloo"></i> Polars GPU Engine</a>
+            <span class="footer-item"> | </span>
+            <a class="footer-item" href="/nx-cugraph"><i class="fa-sharp fa-light fa-chart-network"></i> nx-cuGraph</a>
```

Comment:
I'm questioning the footer layout here. We have "landing pages" for cuvs, cudf.pandas, cudf-polars, and nx-cugraph. But we don't have such pages for every library. Should these be in some kind of top-level menu instead of a footer? This proposal is out of scope for this PR... but probably something we should consider.

Reply:
Agree its a bit awkward - with the docs merge, itll make sense to either remove these pages or move them into the "docs" section which would mean each library has an overview page.